arviz/tests/test_plots.py

```diff
@@ -375,6 +376,17 @@ def test_plot_kde(continuous_model, kwargs):
     assert axes
 
 
+def test_cov():
+    x = np.random.rand(100, 100)
+    assert np.allclose(_cov(x), np.cov(x))
+
+
+def test_stack():
+    x = np.random.randn(100, 100, 5)
+    y = np.random.randn(100, 100, 5)
+    assert np.allclose(_stack(x, y), np.vstack((x, y)))

```

Comment:
`np.allclose` will fail if they both have different shape, but I think it would be more clear if before, the shape was checked.

Reply:
Thank you for pointing this out.